RIP-3 Nft Bank/RIP-3 Nft Bank.md

```diff
@@ -0,0 +1,81 @@
+---
+rip: 3
+title: Nft Bank
+author:  @butterfiolee
+discussions-to: <URL>
+status: Draft
+created: 2022-03-21
+---
+
+## Abstract
+This is a way for users to safely store their nft's outside of their wallet.
+
+## Motivation
+Players over time will accumulate a large amount of assets through the collection of gaming assets as well as other general non-fungible tokens such as art. Wallets as they were originally conceived weren't meant to handle such a large volume of items and eventually they become cumbersome to organize.
+
+To solve this, an Nft Bank is necessary so that users may be able to securely store assets that may not currently be using, thus keeping their wallet manageable.
+
+## Specification
+The Bank contracts will be able to handle the storage of both ERC1155 and ERC721 assets.
+
+Because this bank is a smart contract, it utilizes measures to ensure that if one's private keys are lost, stored assets may still be retrievable. This is achieved by setting backup addresses that will have access to the accounts if the main wallet has not interacted with the contracts for a certain length of time.
+
+Users may also launch their own contracts, which they will have full control over. Though expensive to do, they need not fear that the contracts, and therefore their assets, will be frozen when bugs/security risks are discovered.
```

Comment:
Can you make a few changes here.

We're going to start off with ONLY allowing a user to launch their own contract. We aren't going to create a global bank since it's a central point of failure.